packages/klighd-interactive/src/klighd-interactive-mouselistener.ts

```diff
@@ -77,9 +71,6 @@ export class KeithInteractiveMouseListener extends MoveMouseListener {
     }
 
     mouseDown(target: SModelElement, event: MouseEvent): Action[] {
-        if (this.dserver.connector.editorManager.currentEditor && this.dserver.connector.editorManager.currentEditor.saveable.dirty) {
-            this.dserver.connector.editorManager.currentEditor.saveable.save();
```

Comment:
Has this removed behavior moved somewhere else?

Reply:
Removed. Behavior is not required and should not be implemented in the core packages.

Diagrams are saved after an interaction through the "general/replaceContentInFile" notification from the server. This request is handled by the extension in the file `lsp-handler.ts`.

In my testing on the KEITH master the above code path was never reached. However, I have auto save enabled and I think that it is supposed to save the file before an interaction to prevent problems.
After testing it in VS Code with auto save disabled, everything still works if an interaction occurs while the file is unsaved. VS Code is still able to apply the edits and save the file after an interaction without overwriting the unsaved changes.

Therefore, I think this code is unnecessary.